src/cluster/detail/ArborX_DistributedDBSCANHelpers.hpp

```diff
@@ -287,6 +302,9 @@ void forwardNeighbors(MPI_Comm comm, ExecutionSpace space, Points const &points,
   }
 
   {
+    int comm_rank;
+    MPI_Comm_rank(comm, &comm_rank);
```

Comment:
Why did you add this?

Reply:
Because the corresponding code moved out to `computeRanksTo`.